tests/fsharpqa/Source/CodeGen/EmittedIL/AsyncExpressionStepping/AsyncExpressionSteppingTest6.il.netfx4.bsl

```diff
@@ -127,21 +129,23 @@
                                    int32 x1,
                                    class [FSharp.Core]Microsoft.FSharp.Core.FSharpRef`1<int32> y) cil managed
       {
+        .custom instance void [mscorlib]System.Runtime.CompilerServices.CompilerGeneratedAttribute::.ctor() = ( 01 00 00 00 ) 
+        .custom instance void [mscorlib]System.Diagnostics.DebuggerNonUserCodeAttribute::.ctor() = ( 01 00 00 00 ) 
         // Code size       28 (0x1c)
         .maxstack  8
         IL_0000:  ldarg.0
         IL_0001:  call       instance void class [FSharp.Core]Microsoft.FSharp.Core.FSharpFunc`2<int32,class [FSharp.Core]Microsoft.FSharp.Control.FSharpAsync`1<int32>>::.ctor()
         IL_0006:  ldarg.0
         IL_0007:  ldarg.1
-        IL_0008:  stfld      class [FSharp.Core]Microsoft.FSharp.Control.FSharpAsyncBuilder AsyncExpressionSteppingTest6/AsyncExpressionSteppingTest6/'f3@19-4'::builder@
+        IL_0008:  stfld      class [FSharp.Core]Microsoft.FSharp.Control.FSharpAsyncBuilder AsyncExpressionSteppingTest6/AsyncExpressionSteppingTest6/'f3@19-5'::builder@
         IL_000d:  ldarg.0
         IL_000e:  ldarg.2
-        IL_000f:  stfld      int32 AsyncExpressionSteppingTest6/AsyncExpressionSteppingTest6/'f3@19-4'::x1
+        IL_000f:  stfld      int32 AsyncExpressionSteppingTest6/AsyncExpressionSteppingTest6/'f3@19-5'::x1
         IL_0014:  ldarg.0
         IL_0015:  ldarg.3
-        IL_0016:  stfld      class [FSharp.Core]Microsoft.FSharp.Core.FSharpRef`1<int32> AsyncExpressionSteppingTest6/AsyncExpressionSteppingTest6/'f3@19-4'::y
+        IL_0016:  stfld      class [FSharp.Core]Microsoft.FSharp.Core.FSharpRef`1<int32> AsyncExpressionSteppingTest6/AsyncExpressionSteppingTest6/'f3@19-5'::y
         IL_001b:  ret
-      } // end of method 'f3@19-4'::.ctor
+      } // end of method 'f3@19-5'::.ctor
```

Comment:
The baseline comparer ignored differences between field names, which can differ when the parallel HostedCompiler is used.